File: November/Day8/1544_ Make_The_String_Great.java
class Solution {
    public String makeGood(String s) {
        Stack<Character> st=new Stack<>();
        for(char c:s.toCharArray()){
            if(!st.isEmpty() && Math.abs(st.peek()-c)==32 ){
                st.pop();
            }
            else
                st.push(c);
            
        }
        String p="";
        while(!st.isEmpty()){
            p=st.pop().toString()+p;
        }
        return p;
    }
}

File: November/Day8/1544_ Make_The_String_Great.cpp
class Solution {
public:
    string makeGood(string s) {
     stack<char>st;
     for(int i=0;i<s.size();i++)
     {
         if(st.empty())
          {
             st.push(s[i]);
             continue;
          }
          if(abs(st.top()-s[i]) == 32)
          {
              st.pop();
          }
         else
         {
             st.push(s[i]);
         }
         }
        string ans;
        while(!st.empty())
        {
            ans=st.top()+ans;
            st.pop();
        }
        return ans;
     }
};